udata/templates/macros/metadata.html

```diff
@@ -12,7 +12,7 @@
 <meta name="description" content="{{ description }}" />
 <meta property="og:description" content="{{ description }}" />
 <meta property="og:image" content="{{ image }}" />
-<meta name="keywords" content="{{ ', '.join(current_site.keywords + (keywords)) }}" />
+<meta name="keywords" content="{{ ', '.join(current_site.keywords + keywords) }}" />
```

Comment:
Be careful here, did you try both cases where there is a list and a unique item for `keywords`?